websvc.metro.lib/src/org/netbeans/modules/websvc/metro/lib/metro.xml

```diff
@@ -36,13 +36,23 @@
     </volume>
     <volume>
         <type>javadoc</type>
-        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/metro-1.4-doc.zip!/</resource>
+        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/webservices-api-javadoc.jar!/</resource>
+        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/webservices-extra-javadoc.jar!/</resource>
+        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/webservices-extra-api-javadoc.jar!/</resource-->
+        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/webservices-rt-javadoc.jar!/</resource>
+        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/webservices-tools-javadoc.jar!/</resource>
     </volume>
     <properties>
         <!-- please check with mkleint@netbeans.org before/after updating this or "classpath" section -->
         <property>
             <name>maven-dependencies</name>
-            <value>org.glassfish.metro:webservices-rt:2.3:jar</value>
+            <value>
+                org.glassfish.metro:webservices-api:2.3.1:jar 
+                org.glassfish.metro:webservices-extra:2.3.1:jar
+                org.glassfish.metro:webservices-extra-api:2.3.1:jar 
+                org.glassfish.metro:webservices-rt:2.3.1:jar 
+                org.glassfish.metro:webservices-tools:2.3.1:jar
+            </value>
```

Comment:
lgtm, however i don't recall anymore why you should check with me on this one ;)

I suppose when added to a maven project, all dependencies are properly added to pom.xml  and the project builds?

maybe as part of the PR, remove the reference to me.

Reply:
Thanks, done.